src/Microsoft.SqlTools.Credentials/project.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "Microsoft.SqlTools.Credentials",
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true,
+    "embed": [
+      "../Microsoft.SqlTools.ServiceLayer/Localization/sr.es.resx",
+      "../Microsoft.SqlTools.ServiceLayer/Localization/sr.resx"
+    ],    
+    "compile": {
+      "exclude": "*.*",
```

Comment:
Generally speaking, I'm pretty sure project.json files are meant to be a blacklist versus a whitelist of what files to compile.